mets_adapter/src/main/scala/uk/ac/wellcome/mets/Main.scala

```diff
@@ -0,0 +1,46 @@
+package uk.ac.wellcome.mets
```

Comment:
Slightly being annoying here, but wouldn't it be better if the package contained the word adapter i.e. something like `uk.ac.wellcome.adapter.mets`? We probably don't do it in the sierra adapter cause it's made of so many bits and bobs but I don't feel `mets` is enough to namespace as there is a mets transformer as well

Reply:
:+1: will rename to `mets_adaptor` so it is the same as the directory name. I think using `adaptor` would be confusing unless we do for our other adaptors too.

Reply:
Sounds good to me 👍

Reply:
Just realised forgot to do this, doing now